src/background_jobs/unconfirmed_emails.js

```diff
@@ -25,22 +24,23 @@ module.exports.remove_unconfirmed_email_users = async () => {
             {
                 confirmed_email: false,
             },
-            attributes: ['id'],
+            attributes: ['id', 'createdAt'],
             raw: true,
         },
     )
-
+  
     if (found_users != undefined && found_users.length) {
 
-        // at this stage found_users will look like [{id:1}, {id:2}]
-        // we want to turn it into an ids array which will look like [1,2]
+        // at this stage found_users will look like [ { id: 1, createdAt: '2022-03-26 22:23:43.002 +00:00' } ]
+        // we want to turn it into an ids array which will look like [1], if the createdAt refers to a date before a month ago
         const ids = [];
         found_users.forEach((element) => {
-            ids.push(element.id)
+            if (Date.parse(element.createdAt) < (Date.now() - 30 * 24 * 60 * 60 * 1000)) {
```

Comment:
This should be a constant. It's always a good idea to not hardcode values like these, as you might want to change them later.

Reply:
Hey @supernova350,

I just made it like that ( not a constant ), because first of all, this should be calculated at the time of the execution. Also, I did not use it anywhere else. Just a personal preference.

_What you think ?_

Thanks in advance.
Massiles.

Reply:
Well, calculating `24 * 60 * 60...` does not need to be done at runtime. If you have a constant, for example, `const ACCOUNT_MAX_UNCONFIRMED_AGE_DAYS = 30;`, it just makes the the numbers easier to understand. No problem though, just a suggestion.